twine/commands/upload.py

```diff
@@ -261,16 +141,12 @@ def main(args):
              "a .asc file to include an existing signature with the "
              "file upload.",
     )
-    parser.add_argument(
-        "--verbose",
-        action="store_true",
-        help="Show verbose output."
-    )
 
     args = parser.parse_args(args)
+    upload_settings = settings.Settings.from_argparse(args)
```

Comment:
The change to the environment variable test in `tests/test_upload.py` stems from this. If instead we created the Settings object inside the `upload` function the one invocation would not error but the first would fail (since we'd never call `Settings.from_argparse`. In short, this indicates to me that the environment var test needs to be rewritten to accommodate both of these cases and should be moved to `tests/test_settings.py`